### PR TITLE
feat: update group table

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
[WEB-1739]

## Test Plan
In Groups tab

Check if menu item label from “Edit” to “Edit Group”
Check if menu item label from “Delete” to “Delete Group”
Check if menu item color for “Delete Group” to destructive action red

Check if group level column header copy from “Users” to “Members”

Check if using nameplate layout table to be one “User” column that matches the nameplate style formatting of the user column in the “Users” tab table. with alpha sort

Add confirmation alert for “Remove user from group”

cannot find easy solution 

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

[WEB-1739]: https://hpe-aiatscale.atlassian.net/browse/WEB-1739?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/pages/Admin/RemoveUserFromGroupModal.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      size="small"
+      submit={{
+        handleError,
+        handler: handleOk,
+        text: 'Remote User',
+      }}
+      title="Confirm Delete User from Group">
+      <div>
+        Are you sure you want to remove `{user?.username ?? 'this user'}` from `
+        {groupResult.group.name ?? 'this group'}`?
```
Comment (gt2345) [inline webui/react/src/pages/Admin/RemoveUserFromGroupModal.tsx:60]: Screenshot 2023-10-18 at 4 28 20 PM
The appearance looks a bit odd to me, can we remove the ``? 
Also the title should be `Confirm Removing User from Group`?

**Review by gt2345 — COMMENTED** [review]
The ticket mentioned about users should be alphabetically sorted, not sure if that's implemented

**Comment by keita-determined** [comment]
> The ticket mentioned about users should be alphabetically sorted, not sure if that's implemented

Good catch. I forgot to implement it. Will fix it soon

File: webui/react/src/pages/Admin/RemoveUserFromGroupModal.tsx (hunk 1 above)
Comment (keita-determined) [reply webui/react/src/pages/Admin/RemoveUserFromGroupModal.tsx]: fixed

File: webui/react/src/pages/Admin/RemoveUserFromGroupModal.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  }, [groupResult, onRemoveUser, user]);
+
+  return (
+    <Modal
+      danger
+      size="small"
+      submit={{
+        handleError,
+        handler: handleOk,
+        text: 'Remote User',
```
Comment (gt2345) [inline webui/react/src/pages/Admin/RemoveUserFromGroupModal.tsx:55]: typo: Remove User

Comment (keita-determined) [reply webui/react/src/pages/Admin/RemoveUserFromGroupModal.tsx]: fixed

**Pull request merged**